src/HealthChecks.UI.Core/UIHealthReport.cs

```diff
@@ -19,7 +19,7 @@ public UIHealthReport(Dictionary<string, UIHealthReportEntry> entries, TimeSpan
             TotalDuration = totalDuration;
         }
 
-        public static UIHealthReport CreateFrom(HealthReport report)
+        public static UIHealthReport CreateFrom(HealthReport report, bool hideExceptions = false)
```

Comment:
```suggestion
        public static UIHealthReport CreateFrom(HealthReport report, Func<Exception, string>? exceptionMessage = null)
```

Reply:
> 1. Tests are welcome.

I've added a test for `IHealthReport.CreateFrom()`.
I decided not to add a test for `UIResponseWriter.WriteHealthCheckUIResponseNoExceptionDetails()` because of the `HttpContext`.

Is it OK?

Reply:
Done.